tests/test_patch_dc2.py

```diff
@@ -0,0 +1,161 @@
+import os
+import unittest
+from unittest.mock import patch
+
+from lsst.dc2gen3 import patch_dc2
+
+TEST_CALIB_OLD_ROOT = "/tmp/test_patch_dc2_old_calib_root"
```

Comment:
Given the mocking, I'm not sure if these paths really get used, but in general `tempfile` is preferred over using `/tmp`.